00_Base/src/util/OcpiLocationsUtil.ts

```diff
@@ -0,0 +1,65 @@
+import { Service } from 'typedi';
+import { ChargingStationVariableAttributes } from '../model/variableattributes/ChargingStationVariableAttributes';
+import { OcpiEvse } from '../model/OcpiEvse';
+import { OcpiConnector } from '../model/OcpiConnector';
+import { TEMPORARY_CONNECTOR_ID } from '../model/DTO/ConnectorDTO';
+import { UID_FORMAT } from '../model/DTO/EvseDTO';
+import { OcpiEvseRepository } from '../repository/OcpiEvseRepository';
+import { OcpiConnectorRepository } from '../repository/OcpiConnectorRepository';
+
+@Service()
+export class OcpiLocationsUtil {
+  constructor(
+    private ocpiEvseRepository: OcpiEvseRepository,
+    private ocpiConnectorRepository: OcpiConnectorRepository,
+  ) {}
+
+  public async createOcpiEvsesInfoMap(
+    chargingStationAttributesMap: Record<
+      string,
+      ChargingStationVariableAttributes
+    >,
+  ): Promise<Record<string, OcpiEvse>> {
+    const ocpiEvseMap: Record<string, OcpiEvse> = {};
```

Comment:
Could we have used the `Map` type instead of `Record`?

Reply:
yup map could've worked here, i'm just used to javascript land where we can add straight to empty objects 😅 perhaps another thing we can decide on (always use maps or records or decide on situations where one is better than the other)